bootstrap-joining-demo/kubernetes-api/src/main/resources/application.conf

```diff
@@ -6,8 +6,13 @@ akka {
 
   discovery {
     # pick the discovery method you'd like to use:
-//    method = akka-dns
+    # method = akka-dns
     method = kubernetes-api
+
+    kubernetes-api {
+      pod-label-selector = "app=appka"
+      pod-port-name = "http"
```

Comment:
Rather leave this out -- i.e. the defaults here for consistency with another project Lightbend is working on.

Reply:
you mean to remove the pod-port-name? The latter part of the sentence had even me confused...

Reply:
Sorry for being unclear - Yes, defaults should be sufficient, i.e. no need to declare `pod-port-name`.